Source/WTF/wtf/Assertions.h

```diff
@@ -540,7 +541,11 @@ template<int s, int t> struct check_size {
 #define RELEASE_LOG_STACKTRACE(channel) WTFReleaseLogStackTrace(&LOG_CHANNEL(channel))
 #endif
 
+#ifdef __CHERI__
```

Comment:
`#if __has_feature(capabilities)` is what we generally use for detecting CHERI, or, if you don't care about hybrid, `#ifdef __CHERI_PURE_CAPABILITY__`. Not sure if we actually document that `__CHERI__` even exists anywhere, though we don't document `__CHERI_PURE_CAPABILITY__` either in TR 947 as far as I can see, which is a bit of an oversight...

Reply:
To be honest, I used `__CHERI__` because I'd seen it before, and observed that it's set when I wanted it to be. This port of WebKit actually uses both, so I'll make another change to use `__has_feature(capabilities)` consistently.

JavaScriptCore should work for hybrid and I'd like to preserve that, though cheribuild currently doesn't support it, so it's untested here.

Reply:
See #12.